src/components/select/_select.scss

```diff
@@ -279,8 +279,18 @@
     pointer-events: none;
   }
 
-  &[data-invalid] ~ .#{$prefix}--select__arrow {
-    bottom: 2.5rem;
+  &[data-invalid] ~ {
+    .#{$prefix}--select__arrow {
+      bottom: 2.4rem;
+    }
+
+    .#{$prefix}--select__invalid {
+      position: absolute;
+      right: rem(34px);
```

Comment:
Where does the `34px` value come from? Is it unique for the arrow basically?

Reply:
yeah pretty much, since we are using the glyph icon instead of a standard 16x16 icon for the chevron